addons/crm/models/crm_lead.py

```diff
@@ -764,6 +761,53 @@ def action_schedule_meeting(self):
         }
         return action
 
+    def action_send_mail(self):
+        self.ensure_one()
+        template = self.env.ref('crm.email_template_opportunity_mail')
+        ctx = {
+            'default_composition_mode': 'comment',
+            'default_use_template': bool(template),
+            'default_template_id': template.id,
+        }
+        return {
+            'name': _('Send Email'),
+            'type': 'ir.actions.act_window',
+            'view_type': 'form',
+            'view_mode': 'form',
+            'res_model': 'mail.compose.message',
+            'binding_model': "crm.lead",
+            'binding_views': "list",
+            'id': "crm.action_lead_mass_mail",
+            'target': 'new',
+            'context': ctx,
+        }
+
+    def action_send_sms(self):
+        self.ensure_one()
+        if not self.phone:
+            raise UserError(_("There is no professional mobile for this employee."))
+        context = dict(self.env.context)
+        context.update(default_composition_mode="comment", default_mass_keep_log=True, default_res_id=self.id, default_res_model="crm.lead")
+        return {
```

Comment:
Well this should be located in crm_sms module as crm doe not depend on SMS. SMS is auto install but adds bridge modules. It is not directly integreated in crm.